pennylane/fourier/spectrum.py

```diff
@@ -0,0 +1,261 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains a QNode transform that computes the frequency spectrum of a quantum
```

Comment:
```suggestion
"""Contains a transform that computes the frequency spectrum of a quantum
```
The taxonomy of transforms (what used to be "qnode transforms" in particular) is currently being overhauled and is in flux (see #1315 and #1362), so recommend to not specify a type of transform for now.